include/dsVideoDevice.h

```diff
@@ -307,8 +307,8 @@ dsError_t dsForceDisableHDRSupport(intptr_t handle, bool disable);
  * For source devices, this function returns dsERR_OPERATION_NOT_SUPPORTED always.
  *
  * @param[in] handle    - The handle returned from the dsGetVideoDevice() function
- * @param[in] frfmode   - integer with corresponding Framerate value. 
- *                               Please refer ::dsVideoFrameRate_t for max and min framerate.
+ * @param[in] frfmode   - integer value to enable/disable Auto Framerate mode.
```

Comment:
is it possible to use bool rather integer?

Reply:
It sets the westeros command.
https://gerrit.teamccp.com/plugins/gitiles/rdk/components/generic/devicesettings/soc/amlogic/common/+/24Q3_sprint/dsVideoDevice.c#452

Better to use int